docs/src/prototypes/CopyToClipboard/NotificationProvider.tsx

```diff
@@ -28,9 +35,21 @@ export const NotificationProvider: React.FC = props => {
 
   return (
     <>
-      <Portal open={!!notification}>
-        <Notification>{notification}</Notification>
-      </Portal>
+      {!!targetRef ? (
+        !!notification && (
+          <Tooltip
+            content={notification}
+            pointing={true}
+            open={!!notification}
+            target={targetRef.current}
```

Comment:
This is dangerous, let's use `target`:

```suggestion
            target={targetRef}
```

Reply:
wouldn't that be inconsistent with Popup component where we have
```
target?: HTMLElement
```

Reply:
Oops, wrong suggestion. It should be the same as in `Popup` and be:

```tsx
target={target}
```

So we will pass `target` as `HTMLElement` instead of refs.

***

> Keep in mind that useRef doesn’t notify you when its content changes. Mutating the .current property doesn’t cause a re-render. If you want to run some code when React attaches or detaches a ref to a DOM node, you may want to use a callback ref instead.

https://reactjs.org/docs/hooks-reference.html#useref